.vscode/launch.json

```diff
@@ -170,7 +170,7 @@
             "type": "cortex-debug",
             "request": "launch",
             "cwd": "${workspaceRoot}/examples/${input:mbedApp}/mbed",
-            "executable": "./build-CY8CPROTO_062_4343W/${input:mbedFlashProfile}/chip-mbed-${input:mbedApp}-example.hex",
+            "executable": "./build-CY8CPROTO_062_4343W/${input:mbedFlashProfile}/chip-mbed-${input:mbedApp}-example.elf",
```

Comment:
Why do we change used executable format here? 
Is there any benefits by switching to .elf?
Did you compare both solutions?